src/code/models/simulation.coffee

```diff
@@ -7,92 +7,89 @@ scaleInput = (val, nodeIn, nodeOut) ->
-        count--
-        return
-      sourceNode = link.sourceNode
-      inV = if sourceNode.previousValue? then sourceNode.previousValue else sourceNode.initialValue
-      inV = scaleInput(inV, sourceNode, this)
-      outV = @previousValue or @initialValue
-      nextValue = link.relation.evaluate(inV, outV, link.sourceNode.max, @max)
-      value += nextValue
-    if count >= 1
-      value = value / count
-    else
-      value = @previousValue or @initialValue       # we had no defined inbound links
+  # collectors aren't calculated in this phase, but they do capture initial/previous values
+  startValue = if @previousValue? then @previousValue else @initialValue
+  return startValue if @isAccumulator or count < 1
+
+  value = 0
+  inValues = []
+
+  _.each links, (link) =>
+    return unless link.relation.isDefined
+    sourceNode = link.sourceNode
+    inV = if sourceNode.previousValue? then sourceNode.previousValue else sourceNode.initialValue
+    inV = scaleInput(inV, sourceNode, this)
+    outV = startValue
+    inValues.push link.relation.evaluate(inV, outV, link.sourceNode.max, @max)
+
+  # for now, if any a node points to a collector, it should use the scaled product
+  # ultimately this should be a default which can be overridden by the user, in
+  # which case the setting would presumably become part of the node model
+  useScaledProduct = !!(_.find @outLinks(), (link) -> link.targetNode.isAccumulator)
+  value = combineInputs(inValues, useScaledProduct)
 
   # if we need to cap, do it at end of all calculations
   value = @filterFinalValue value
 
   value
 
 SetAccumulatorValueFunction = (nodeValues) ->
-  # get add, subtract and transfer links
-  links = @inLinks('accumulator').concat(@inLinks('transfer'))
+  # collectors only have accumulator and transfer links
+  links = @inLinks('accumulator').concat(@inLinks('transfer')).concat(@outLinks('transfer'))
   return unless links.length > 0
 
-  value = 0
+  linkScaleFactor = 100
+
+  deltaValue = 0
   for link in links
     {sourceNode, relation, transferNode} = link
     inV = nodeValues[sourceNode.key]
     switch relation.type
       when 'accumulator'
         # TODO: check if outV is really equal to inV here - I'm not clear what outV is in this case'
-        value += relation.evaluate(inV, inV, sourceNode.max, @max)
+        deltaValue += relation.evaluate(inV, inV, sourceNode.max, @max) / linkScaleFactor
 
       when 'transfer'
-        # TODO: check with Dan if the transfer value really is meant to be a percentage
         transferValue = nodeValues[transferNode.key]
-        transferPercentage = transferValue / Math.max(transferNode.max, transferValue)
-        for modifierLink in transferNode.inLinks('transfer-modifier')
-          transferPercentage *= modifierLink.relation.evaluate(inV, inV, sourceNode.max, transferNode.max)
+        # transfer values are scaled unless they have a modifier
+        if not transferNode.inLinks('transfer-modifier').length
```

Comment:
Yes, since `transfer-modifier` links can interact with `range` links on a transfer node, they're handled in the first pass. So `range` and `transfer-modifier` links are handled in the first pass, while `accumulator` and `transfer` links are handled in the second.